dietpi/dietpi-software

```diff
@@ -8722,6 +8722,8 @@ SocksPort 0
 ControlSocket 0
 _EOF_
 		elif [[ $G_WHIP_RETURNED_VALUE == 'Exit' ]]; then
+			G_AGI unbound
```

Comment:
Would it make more sense to call `dietpi-software` installation for `unbound` ?

Reply:
While that probably would be preferred, I'm not sure how to do that from the configuration step.

Reply:
`dietpi-software` must not be called from within `dietpi-software`, so if the unbound install step is nothing else than the APT package install and the unbound config step comes below the Tor relay config step, it is fine as it is.

But if I am right, the unbound config step currently comes before these, around line 8400. So currently, installing the package here potentially breaks existing Pi-hole installs and other DNS servers running at port 53. And does Tor use it automatically when it is there, since I don't see it being configured currently?

Reply:
What I meant was to use the already existing unbound installation and not to fire it up using `apt`. What about changing the order and move `TOR Relay` before `unbound` inside `dietpi-softwarw`

Reply:
You're right, it does need to be configured.  TOR uses the DNS server being used by the system.